### PR TITLE
Header fixes

### PR DESCRIPTION
**Pull request by GusBaamonde**
fix: add reset buttons
feat: add header messages
BREAKING CHANGE: add "button" element instead "link" element in user-menu

### CODE REVIEW

File: assets/scss/core/_reset.scss [hunk 1]
```diff
@@ -159,6 +160,14 @@ img {
 }
 /* Forms
    ========================================================================== */
+
+button {
```
Comment (fcoronelmakingsense) [inline assets/scss/core/_reset.scss]: maybe this add in a css class

File: assets/scss/templates/_messages-header.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/* messages headers*/
+
+.messages-container {
+  display: flex;
+  // height: 50px;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_messages-header.scss]: remove comment

File: assets/scss/templates/_messages-header.scss [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  width: 100%;
+}
+
+.messages-container p {
+  color: #fff;
+  font-size: $dp-font-size-small;
+}
+
+.messages-container.error .button {
+  color: #e34f45;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_messages-header.scss]: maybe put in a variable

File: assets/scss/templates/_messages-header.scss [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+.messages-container.message .button {
+  color: #65c192;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_messages-header.scss]: maybe put in a variable

File: assets/scss/templates/_messages-header.scss [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  background: $dp-color-yellow;
+}
+
+.messages-container.error {
+  background: $dp-color-red;
+}
+/* buttons */
+.button {
+  border: 0;
+  border-radius: 3px;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_messages-header.scss]: this is not default value?

File: assets/scss/templates/_messages-header.scss [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  color: #fff;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_messages-header.scss]: use color white variable

File: assets/scss/templates/_messages-header.scss [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+  line-height: 45px;
+  padding-left: 20px;
+  padding-right: 20px;
+  height: 45px;
+  text-align: center;
+  transition: all 0.3s ease;
+}
+
+.button--light {
+  background: #fff;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_messages-header.scss]: use color white variable

File: assets/scss/templates/_messages-header.scss [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+  border: 1px solid #fff;
+  color: #666;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_messages-header.scss]: use color variable

**Comment by andresmoschini** [comment]
test this please (this message is for Jenkins to build again)

**Comment by andresmoschini** [comment]
>  BREAKING CHANGE: add "button" element instead "link" element in user-menu
Did you remove support for _link_ element? if not, it is not a breaking change.

**Pull request merged**